src/util.ts

```diff
@@ -954,3 +954,23 @@ export function runGodoc(cwd: string, packagePath: string, receiver: string, sym
 		});
 	});
 }
+
+/**
+ * Returns the alternate language server being used
+ * @param goConfig Workspace configuration
+ */
+export function getAlternateLanguageServer(goConfig: vscode.WorkspaceConfiguration): string {
+	if (goConfig['useLanguageServer'] === true
+		&& goConfig['alternateTools']
+		&& goConfig['alternateTools']['go-langserver']
```

Comment:
do we need to define this in the package.json file?

Reply:
If we do, then we need to define all the tools used in the extension as well.